File: Coding Ninjas Studio/Coding World Cup/2023/Day 36 - NZ vs SL - Support Players.cpp
#include <bits/stdc++.h> 
string stringOperation(string &s, int k){
    // Write your code here
    int operations = 0;
    
    for (char &c : s) {
        if (c != 'a' && c != 'e' && c != 'i' && c != 'o' && c != 'u') {
            operations++;
        } else {
          continue;
        }
    }

    return (operations == k) ? "YES" : "NO";
}

File: Coding Ninjas Studio/Coding World Cup/2023/Day 36 - NZ vs SL - Support Players.java
import java.util.*;
import java.io.*;

public class Solution {
    static String stringOperation(String s, int k) {
        // Write your code here.
        Set<Character> vowels = new HashSet<>(Arrays.asList('a', 'e', 'i', 'o', 'u'));
        int changesNeeded = 0;

        for (int i = 0; i < s.length(); i++) {
            if (!vowels.contains(s.charAt(i))) {
                changesNeeded++;
            }
        }

        if (changesNeeded <= k && k <= s.length()) {
            return "YES";
        } else {
            return "NO";
        }
    }
}